README.txt

```diff
@@ -1,162 +1,89 @@
+INTRODUCTION
+------------
 
-= Vote Up/Down =
+The Vote Up/Down module allows the user to cast votes on some entities in
+a Drupal site with different widgets. This module uses Voting API to implement
+votes and Chaos Tools to provide AJAX support and make custom widgets.
+ * For a full description of the module, visit the project page:
+   https://drupal.org/project/vote_up_down
```

Comment:
Let's use current host of d.o, i.e. https://www.drupal.org.

Also  a newline seems to be needed here, see the rendered output on https://github.com/sudhanshu16/vote_up_down/blob/8.x-1.x/README.txt